automation/JJB/golang_analysis.yaml

```diff
@@ -1,5 +1,5 @@
 - job:
-          name: golang_coverage
+          name: golang_coverage_analysis
```

Comment:
Slight nit: I recommend a consistant naming scheme, so this would be "sonarqube_golang_analysis".